include/state-observation/dynamics-estimators/kinetics-observer.hpp

```diff
@@ -418,7 +482,11 @@ class STATE_OBSERVATION_DLLAPI KineticsObserver : protected DynamicalSystemFunct
   ///
   /// @param localKinematics
   /// @return Kinematics
-  Kinematics getKinematicsOf(const Kinematics & localKinematics) const;
+  LocalKinematics getLocalKinematicsOf(const LocalKinematics & localKinematics) const;
+  /*
+  Kinematics getGlobalKinematicsOf(const LocalKinematics & localKinematics) const;
+  */
+  Kinematics getGlobalKinematicsOf(const Kinematics & kin) const;
```

Comment:
I wrote them this way for the case we use this Kinematics or LocalKinematics in a kinematics composition. Using a different name would allow to make sure if we are using local or global kinematics, but it is maybe unnecessary.